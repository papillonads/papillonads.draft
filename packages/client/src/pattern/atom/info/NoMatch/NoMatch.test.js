import noMatchDefault from './NoMatch.render'

describe('<NoMatch />', () => {
  describe('Snaphot', () => {
    test('must match noMatchDefault', () => {
      expect(global.renderToJSON(noMatchDefault)).toMatchSnapshot()
    })
  })
})
