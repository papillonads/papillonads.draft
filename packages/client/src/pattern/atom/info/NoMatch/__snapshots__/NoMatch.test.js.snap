// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NoMatch /> Snaphot must match defaultRender 1`] = `null`;
